src/component/StaticContent/StaticContent.js

```
@@ -1,80 +1,162 @@
+              : pathname === "blog"
+              ? "Blog"
+              : ""}
+          </h3>
+        </div>
+        <div className="PrivacyPolicyArea">
+          <div className="VerticalTabs">
+            <div className="row">
+              <div className="col-md-12">
+                <div className="TabCommonBox TermsConditionsContent">
+                  <div
+                    dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)